php_phongo.c

```diff
@@ -602,7 +602,7 @@ static int phongo_execute_parse_options(mongoc_client_t* client, int server_id,
 				if (!process_read_concern(*driver_option, mongoc_opts TSRMLS_CC)) {
 					return false;
 				}
-			} else if ((!strcasecmp(string_key, "readPreference")) && (type == PHONGO_COMMAND_READ)) {
```

Comment:
In https://github.com/mongodb/mongo-php-driver/commit/eea07a062e638b61e0d4043749da598a964b4545#diff-c06c6e1c9374aecabcf544157f9d0c26R564, a change was made to the PHP 7 code to check for `PHONGO_COMMAND_RAW` but was missed for PHP 5.